src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolJudicialRoleMappingSSCS.java

```diff
@@ -0,0 +1,419 @@
+        QueryResults queryResults = (QueryResults) results.getValue(ROLE_ASSIGNMENTS_RESULTS_KEY);
+        for (QueryResultsRow row : queryResults) {
+            roleAssignments.add((RoleAssignment) row.get("$roleAssignment"));
+        }
+
+        //assertion
+        assertFalse(roleAssignments.isEmpty());
+        assertEquals(1, roleAssignments.size());
+        assertEquals("caseworker-sscs-disability-feepaid",roleAssignments.get(0).getRoleName());
+        assertEquals(judicialOfficeHolders.stream().iterator().next().getUserId(),roleAssignments.get(0).getActorId());
+        assertEquals("Fee-Paid", roleAssignments.get(0).getAttributes().get("contractType").asText());
+
+    }
+
+    @Test
+    void shouldReturnTribunalServiceFeePaid() {
+
+        judicialOfficeHolders.forEach(joh -> joh.setOffice("SSCS Tribunal Member Service-Fee Paid"));
+
+        //Execute Kie session
+        buildExecuteKieSession(getFeatureFlags("sscs_wa_1_0", true));
+
+        //Extract all created role assignments using the query defined in the rules.
+        List<RoleAssignment> roleAssignments = new ArrayList<>();
+        QueryResults queryResults = (QueryResults) results.getValue(ROLE_ASSIGNMENTS_RESULTS_KEY);
+        for (QueryResultsRow row : queryResults) {
+            roleAssignments.add((RoleAssignment) row.get("$roleAssignment"));
+        }
+
+        //assertion
+        assertFalse(roleAssignments.isEmpty());
+        assertEquals(1, roleAssignments.size());
+        assertEquals("caseworker-sscs-disability-feepaid",roleAssignments.get(0).getRoleName());
+        assertEquals(judicialOfficeHolders.stream().iterator().next().getUserId(),roleAssignments.get(0).getActorId());
+        assertEquals("Fee-Paid", roleAssignments.get(0).getAttributes().get("contractType").asText());
+
+    }
+
+    @Test
+    void shouldReturnTribunalFinanciallyQualifiedRole() {
+
+        judicialOfficeHolders.forEach(joh -> joh.setOffice("SSCS Tribunal member financially qualified"));
+
+        //Execute Kie session
+        buildExecuteKieSession(getFeatureFlags("sscs_wa_1_0", true));
+
+        //Extract all created role assignments using the query defined in the rules.
+        List<RoleAssignment> roleAssignments = new ArrayList<>();
+        QueryResults queryResults = (QueryResults) results.getValue(ROLE_ASSIGNMENTS_RESULTS_KEY);
+        for (QueryResultsRow row : queryResults) {
+            roleAssignments.add((RoleAssignment) row.get("$roleAssignment"));
+        }
+
+        //assertion
+        assertFalse(roleAssignments.isEmpty());
+        assertEquals(1, roleAssignments.size());
+        assertEquals("caseworker-sscs-financial-feepaid",roleAssignments.get(0).getRoleName());
+        assertEquals(judicialOfficeHolders.stream().iterator().next().getUserId(),roleAssignments.get(0).getActorId());
+        assertEquals("Fee-Paid", roleAssignments.get(0).getAttributes().get("contractType").asText());
+
+    }
+
+
```

Comment:
Can you please add test for 
1. Invalid authorisation(expired enddate) and valid appointment
2. Invalid authorisation(wrong servicecode) and valid appointment
3. Invalid authorisation(expired enddate) and Invalid appointment(wrong servicecode)
4. Valid appointmentRoles
5. Invalid appointmentRoles